src/content/en/updates/2017/06/credential-management-updates.md

```diff
@@ -0,0 +1,333 @@
+
+In the following screenshot, credential information stored under `login.aliexpress.com`
+is visible to `m.aliexpress.com` and available for the user to choose from:
+
+<figure>
+  <img src="/web/updates/images/2017/06/credentials.png"
+       alt="Account chooser showing selected subdomain login details"/>
+</figure>
+
+<aside>
+  <strong>Coming soon:</strong>
+  Sharing credentials between totally different domains is in development.
+</aside>
+
+## Chrome 60
+
+Chrome 60 introduces several important changes to the
+[Credential Management API](/web/fundamentals/security/credential-management):
+
+* `PasswordCredential` object now includes a password.
+
+*  As the custom `fetch()` function is no longer required to fetch the password,
+it will be deprecated soon.
+
+* `navigator.credentials.get()` now accepts an enum `mediation`
+instead of boolean flag `unmediated`.
+
+* `requireUserMediation()` renamed to `preventSilentAccess()`.
+
+*  New method `navigator.credentials.create()`
+asynchronously creates credential objects.
+
+### `PasswordCredential` object now includes password 
+
+The Credential Management API took a conservative approach to handling passwords.
+It concealed passwords from JavaScript, requiring developers
+to send the `PasswordCredential` object directly to their server
+for validation via an extension to the `fetch()` API.
+
+But this approach introduced a number of restrictions.
+We received feedback that developers could not use the API because:
+
+* They had to send the password as part of a JSON object.
+
+* They had to send the hash value of the password to their server.
+
+After performing a security analysis and recognizing that concealing passwords
+from JavaScript did not prevent all attack vectors as effectively as we were hoping,
+we have decided to make a change.
+
+The Credential Management API now includes a raw password
+in a returned credential object so you have access to it as plain text.
+You can use existing methods to deliver credential information to your server:
+
+    navigator.credentials.get({
+
+      password: true,
+
+      federated: {
+
+        provider: [ 'https://accounts.google.com' ]
+
+      },
+
+      mediation: 'silent'
+
+    }).then(c => {
+
+      if (c) {
+
+        let form = new FormData();
+
+        form.append('email', c.id);
+
+        form.append('password', c.password);
+
+        form.append('csrf_token', csrf_token);
+
+        return fetch('/signin', {
+
+          method: 'POST',
+
+          credentials: 'include',
+
+          body: form
+
+        });
+
+      } else {
+
+        // Fallback to sign-in form
+
+      }
+
+    }).then(res => {
+
+      if (res.status === 200) {
+
+        return res.json();
+
+      } else {
+
+        throw 'Auth failed';
+
+      }
+
+    }).then(profile => {
+
+      console.log('Auth succeeded', profile);
+
+    });
+
+### Custom fetch will be deprecated soon
+
+Now that passwords return in the `PasswordCredential` object,
```

Comment:
I might change this to a warning:
`Warning: Now that passwords are no longer returned in the ``PasswordCredential`` object, the custom ``fetch()`` function will stop working in Chrome 62. Developers **must** update their code.`